FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -207,6 +207,7 @@ final class ComposeViewController: TableNodeViewController {
         self.contextToSend.recipients = [
             ComposeMessageRecipient(
                 email: "tom@flowcrypt.com",
+                name: "Tom",
                 type: .to,
                 state: decorator.recipientIdleState
             )
```

Comment:
I don't understand, why is there my email, or any email, in production code? This is a VC right? This has no business being here?

when is this method called?